dev/PushNotifications/externs.h

```diff
@@ -0,0 +1,8 @@
+﻿#pragma once
+#include "pch.h"
+#include "PushNotificationReceivedEventArgs.h"
+
+extern wil::unique_handle g_waitHandleForArgs;
+extern wil::critical_section g_lock;
```

Comment:
Do you require a critical section or just a lock? SRWLOCK is generally recommended these days over critical section (`wil::srwlock`).

Or even better, since you're in Reunion, just use `std::mutex` and not sweat what's the optimal implementation for a lock :-)

Reply:
Resolving since `g_lock` is removed.